### ISSUE TITLE
Add accessibility updates to the Show More/Show Less buttons on hierarchy lists

### ISSUE THREAD

**Issue by kamerynB** [project-lux/lux-frontend#293]
**Problem Description:**
The Show More/Less buttons are missing accessibility features when users are navigating the site with a keyboard on the hierarchy lists.

**Expected Behavior/Solution:**
The focus of the keyboard should move back to the list after a user selects to Show More/Less.

**Requirements:**

- [x] Add focus to the top of the next set of entities displayed after a user hits `Show More`
- [x] Add focus to the bottom of the currently displayed list of entities after a user hits `Show Less`

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] Wireframe/Mockup - Kam
- [x] ~Committee discussions - Sarah~
- [x] ~Feasibility/Team discussion - Sarah~
- [x] ~Backend requirements - TBD~
- [x] Frontend requirements - Kam
- [ ] Are new regression tests required for QA - Amy
- [x] ~Questions~

**UAT/LUX Examples:**

- https://lux-front-tst.collections.yale.edu/view/concept/ea1a14d9-64bb-4d87-943b-ede6ffe72dd0

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
N/A

**Related links:**
N/A

**Wireframe/Mockup:**
Current state when user hits `Show More` with the keyboard:
<img width="350" alt="Screenshot 2024-07-12 at 12 56 05 PM" src="https://github.com/user-attachments/assets/384e36d9-47e0-4462-b341-8de5c1056ee7">

Expected state when a user hits `Show More` with the keyboard:
<img width="350" alt="Screenshot 2024-07-12 at 12 56 17 PM" src="https://github.com/user-attachments/assets/11319897-d6cf-4266-898d-1c1977931dda">

Expected state when a user hits `Show Less` with keyboard:
<img width="350" alt="Screenshot 2024-07-12 at 12 56 34 PM" src="https://github.com/user-attachments/assets/a3219d27-3a39-4719-8787-8e3822c8046a">

**Comment by kamerynB** [comment]
@roamye @prowns This change was reviewed by Kara and approved. Once UAT happens this issue can be closed.

**Comment by roamye** [comment]
Approved by UAT

**Comment by roamye** [comment]
This was approved by Kara and is now in PROD. Closing.
